File: Atividade Avaliativa Final/Javascript/entity/torneio/TorneioFutsal.js
import { Torneio } from './Torneio.js';

export class TorneioFutsal extends Torneio {
  constructor(nome) {
    super(nome);
  }

  getModalidade() {
    return 'Futsal';
  }

  getPosicoes() {
    return ['Goleiro', 'Fixo', 'Ala', 'Pivô'];
  }
}

File: Atividade Avaliativa Final/Python/entity/torneio/torneio_futsal.py
from entity.torneio.torneio import Torneio

class TorneioFutsal(Torneio):
    def get_modalidade(self) -> str:
        return "Futsal"

    def get_posicoes(self) -> list:
        return ["Goleiro", "Fixo", "Ala", "Pivô"]